### DISCUSSION TITLE
Certain template tags cause fatal error for scans with -t switch

### DISCUSSION THREAD

**Discussion by 0xceba** [projectdiscovery/nuclei#1533] [Q&A] [answered]
### Nuclei version:
2.5.9

### Current Behavior:
Nuclei scans with a template specified via the `-t` switch fail to run if the template contains certain tags. The fatal error is caused by templates containing the tags "fuzz" or "misc" or possibly other values.

### Expected Behavior:
The scan should be launched with the desired template.

### Steps To Reproduce:
1. Add "fuzz" to a template's tags.
2. Attempt a scan with the modified template: `nuclei -t path/to/template.yaml`
3. Note the fatal error: `[FTL] Could not run nuclei: no valid templates were found`

### Supporting Material
The following media demonstrates the issue:
![nuclei_fatal_tag_fuzz](https://user-images.githubusercontent.com/44234156/150876199-4d1cad81-70ee-4d96-8ac1-6c55f06da6d7.gif)

**Comment by ehsandeep** [comment, marked as answer]
@0xceba https://github.com/projectdiscovery/nuclei/discussions/1337#discussioncomment-1755851
